node/node.go

```diff
@@ -725,6 +709,25 @@ func (n *Node) Dispatch() error {
 		n.Net.ManuallyTrack(bootstrapper.ID, bootstrapper.IP)
 	}
 
+	// Log a warning if we aren't able to connect to a sufficient portion of
+	// nodes.
+	go func() {
```

Comment:
Super small nit, but I think it would make more sense to move this prior to calling `n.Net.ManuallyTrack`. The comment below here stating `// Start P2P connections` is a bit misleading. Calling `n.Net.Dispatch()` starts handling _inbound_ connections. Calling `ManuallyTrack` can start creating _outbound_ connections.